app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,9 +84,14 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      ems_ref_for_vm_id(import.vm.id)
     end
   end
 
+  def ems_ref_for_vm_id(vm_id)
+    "/api/vms/#{vm_id}"
```

Comment:
@mkanoor this format is taken directly from the database and I imagine the refresh is using exactly those REST paths to reference RHEVM's entities